go.mod

```diff
@@ -1,24 +1,54 @@
 module github.com/cubefs/cubefs-csi
 
-go 1.13
+go 1.18
 
 require (
-	github.com/container-storage-interface/spec v1.1.0
-	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/container-storage-interface/spec v1.5.0
```

Comment:
Since there are no breaking changes from v1.1.0 to v1.5.0, so I just upgraded to v1.5.0 for future maintaince;

**csi-provisioner**  v2.2.2 wrote the doc to support v1.0.0. to v1.4.0 may just because the v1.5.0 was not released that time;

Both of them works well in actual  testing.